### DISCUSSION TITLE
osu!lazer tablet sensitivity support

### DISCUSSION THREAD

**Discussion by RelaxedNate2** [ppy/osu#25855] [Q&A]
Currently on osu!lazer when i select tablet there are no options for changing sensitivity. I have been playing with sensitivity on osu!stable for about 5 years and every time I've tried to switch to area its felt off. Wondering if this is intended or just an unintended mistake of removing the option for tablet sensitivity support. Also to my knowledge i dont have any tablet drivers installed on my computer so disabling the tablet in game makes my tablet not work.

**Comment by peppy** [comment]
If you mean tablet responding to mouse sensitivity, then you can achieve the exact same thing by making your area smaller.
